platform/core/src/services/UIDialogService/index.js

```diff
@@ -51,29 +45,29 @@ function createUIDialogService() {
 /**
  * Show a new UI dialog;
  *
- * @param {DialogProps} props { id, content, onSubmit, onClose, onStart, onDrag, onStop, isDraggable, defaultPosition, position }
+ * @param {DialogProps} props { id, content, contentProps, onStart, onDrag, onStop, isDraggable, showOverlay, defaultPosition, position }
  */
 function create({
-  id,
-  content,
-  onSubmit,
-  onClose,
-  onStart,
-  onDrag,
-  onStop,
-  isDraggable,
-  defaultPosition,
-  position,
+  id = null,
+  content = null,
+  contentProps = null,
+  onStart = null,
+  onDrag = null,
+  onStop = null,
+  isDraggable = true,
+  showOverlay = false,
+  defaultPosition = null,
+  position = null,
```

Comment:
I'm not sure there's much value in defaulting these to `null`. Not a blocker, just confused.

Reply:
Habits when using conditional rendering / error boundaries in React render method since undefined will be an object instead of a null value which is renderable. But! it's not this case though :D so ill remove it, thanks.